content/general/limits.textile

```diff
@@ -352,9 +352,24 @@ h2(#message). Message limits
 Message limits relate to the number, rate and bandwidth of "messages":/channels/messages on your account.
 
 |_=. Limit |_=. Free |_=. PAYG |_=. Business (legacy) |_=. Committed Use |
+| "Server-side batching [beta]":#message-publish-rate | 50 | 50 | 50 | 50 |
 | "Message size":#message-size (KiB) | 16 | 64 | 64 | 256 |
 | "History TTL":#history-ttl (hours) | 24 | 72 | 72 | Custom |
 
+h3(#server-side-batching). Server-side batching [beta]
+
+The "server-side batching":/channels#server-side-batching process works through a *batching policy* and a *batching interval*:
+
+h4. Batching policy
+
+* Each batch can contain up to 50 messages.
```

Comment:
@JoaoDiasAbly Is that the default limit you've implemented? Why so low?

Reply:
As we've discussed this is just a conservative number although it's not backed by any evidence. This number can be changed depending on the data we gather from load testing